app/models/user.rb

```diff
@@ -292,8 +292,8 @@ def remember_me
   end
 
   def cached_followed_tag_names
-    cache_name = "user-#{id}-#{updated_at}/followed_tag_names"
-    Rails.cache.fetch(cache_name, expires_in: 24.hours) do
+    cache_name = "user-#{id}-#{following_tags_count}/followed_tag_names"
```

Comment:
If I followed one tag and then unfollowed another I'd have the same value here. Luckily we have another option which is perfect: `last_followed_at`, which gets touched whenever someone takes a follow action.

I'm not 100% sure this gets touched after an _unfollow_ so you could include both `last_followed_at` and `following_tags_count` to be safe.

Reply:
Well the count will change after an unfollow and if another follow is issued right after then `last_followed_at` changes so I believe we just need `last_followed_at`